### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by gsemet**

### CODE REVIEW

File: docs/source/contributing/dev_env.rst [hunk 1]
```diff
@@ -2,20 +2,49 @@
...
+    $ ./scripts/bootstrap-dev-fedora.sh
+
+Arch Linux
+----------
+
+Execute:
+
+.. code-block:: bash
+
+    $ ./scripts/bootstrap-dev-arch.sh
```
Comment (aichingm) [inline docs/source/contributing/dev_env.rst]: Did I miss something or does this file `./scripts/bootstrap-dev-fedora.sh` not exist?

Comment (gsemet) [reply docs/source/contributing/dev_env.rst]: yes it is on my other branch at home I'll upload it tonight :)

Comment (aichingm) [reply docs/source/contributing/dev_env.rst]: Didn't saw you comment. Forget my second ;)

Comment (gsemet) [reply docs/source/contributing/dev_env.rst]: perfect I'll use it!

Comment (aichingm) [reply docs/source/contributing/dev_env.rst]: :laughing: Ok

**Comment by gsemet** [comment]
@aichingm thanks for your help. Do you think guake works quite properly on Arch for the moment? Do you think I should release 3.2.1?

**Comment by aichingm** [comment]
Yes I think so. I mean there are the issues you know about like quickopen and url links but I don't think that there are more arch specific issues, building works like a charm, and for me it is running without any show-stoppers. Except one which I just found (minor, I will open an issue) by clicking through the preferences window  and testing. Even the 'Start Guake at login' works which I find is awesome!

**Comment by gsemet** [comment]
cool. That's odd because I have quick open and hyperlinks working on fedora and ubuntu. I'll release we'll see that for next version :) thanks for the help!

**Pull request merged**